recipes/_master_base_config.rb

```diff
@@ -28,6 +28,38 @@
 # Get VPC CIDR
 node.default['cfncluster']['ec2-metadata']['vpc-ipv4-cidr-block'] = get_vpc_ipv4_cidr_block(node['macaddress'])
 
+# Get shared_dir path and mount EFS filesystem
```

Comment:
Most of the code is duplicated,  what about creating a new recipe, included by both `_compute_base_config.rb` and `_master_base_config.rb`?

Reply:
Agree, a new efs_mount reciped included both in the master and compute recipes.